src/PresentationalComponents/Battery/battery.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import classNames from 'classnames';
+
+import './battery.scss';
+
+/**
+ * This is a dumb component that only recieves properties from a smart component.
+ * Dumb components are usually functions and not classes.
+ *
+ * @param props the props given by the smart component.
+ */
+
+class Battery extends React.Component {
```

Comment:
No need to use class in here since there are no methods used with lifecycle of component so it might look better using inline function.

Also consider allowing consumer to pass additional `className` and spread `props` in there so consumers can for instance pass their `styles` object, `disabled` and other attributes.
```JSX
const Battery = ({severity, label, labelHidden, className, ...props}) => {
  let severityClass ...
  if ...
}
```